pkg/configuration/http.go

```diff
@@ -0,0 +1,39 @@
+package configuration
+
+import (
+	"context"
+	"net/http"
+
+	commonhttp "github.com/cloudtrust/common-service/v2/http"
+	"github.com/cloudtrust/common-service/v2/log"
+	"github.com/cloudtrust/keycloak-bridge/internal/constants"
+	"github.com/go-kit/kit/endpoint"
+	http_transport "github.com/go-kit/kit/transport/http"
+)
+
+const (
+	prmContextKey = "context-key"
+	prmRealmName  = "realm"
+)
+
+// MakeConfigurationHandler makes an HTTP handler for the configuration endpoint
+func MakeConfigurationHandler(e endpoint.Endpoint, logger log.Logger) *http_transport.Server {
+	return http_transport.NewServer(e,
+		decodeConfigurationRequest,
```

Comment:
You can directly inline decodeConfigurationRequest here.
Ex: https://github.com/cloudtrust/keycloak-bridge/blob/master/pkg/kyc/http.go#L28